test/cpp/microbenchmarks/bm_fullstack_trickle.cc

```diff
@@ -320,8 +320,8 @@ static void BM_PumpStreamServerToClient_Trickle(benchmark::State& state) {
 }
 
 static void StreamingTrickleArgs(benchmark::internal::Benchmark* b) {
-  for (int i = 1; i <= 128 * 1024 * 1024; i *= 8) {
-    for (int j = 64; j <= 128 * 1024 * 1024; j *= 8) {
+  for (int i = 1; i <= 128 * 1024 * 1024; i *= 16) {
```

Comment:
Not needed right now, since we're not really graphing these... but this will cause a data discontinuity. It may be possible to just explicitly call out the values that we're using.